tvm-opcodes/src/main/kotlin/org/ton/bytecode/TvmInstLocation.kt

```diff
@@ -0,0 +1,65 @@
+package org.ton.bytecode
+
+import kotlinx.serialization.Contextual
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+sealed class TvmInstLocation {
```

Comment:
I believe this class should be also opened to overriding such as `TvmCodeBlock`. So, I suggest making it a simple `interface`